src/community/label_propagation.jl

```diff
@@ -104,20 +106,22 @@ function vote!(g::AbstractGraph, m::Vector, c::NeighComm, u::Integer)
             return lbl
         end
     end
+
+    return zero(eltype(c.neigh_pos)) # Case should never occur, just for type stability
```

Comment:
I would rather initialize a new variable before the loop, set it to `lbl` inside the loop and `break` at line 106, returning the new variable.